apps/accounts/api/automations/change_secret_dashboard.py

```diff
@@ -152,7 +152,10 @@ def get(self, request, *args, **kwargs):
             if ongoing_counts is None:
                 execution_ids = []
                 inspect = app.control.inspect()
-                active_tasks = inspect.active()
+                try:
+                    active_tasks = inspect.active()
+                except Exception:
+                    active_tasks = None
                 if active_tasks:
                     for tasks in active_tasks.values():
                         for task in tasks:
```

Comment:
The code appears to have no significant issues, but there could be a minor redundancy in the usage of "tasks" when printing items inside loop because it can also represent multiple elements. Instead of using single string concatenation `"["` and `"]"`, you can use a comma-separated list with square brackets instead.

Here's an updated version:

```python
for task in active_tasks.get('values', []).get("tasks", []):
    # Print item(s) here

For example, if you need to print all values under 'keys' key from dictionary-like structure, replace:

```python
   for tasks in active_tasks:
        for task in tasks['tasks']
```

With this:

```python 
   for tasks in enumerate(active_tasks):  
       for task_details in tasks[1]['tasks'] :
            #Print item here
```